numpy/distutils/misc_util.py

```diff
@@ -2296,23 +2296,12 @@ def generate_config_py(target):
     f.write("""
 
 import os
-import sys
 
 extra_dll_dir = os.path.join(os.path.dirname(__file__), '.libs')
 
-if os.path.isdir(extra_dll_dir) and sys.platform == 'win32':
-    try:
-        from ctypes import windll, c_wchar_p
-        _AddDllDirectory = windll.kernel32.AddDllDirectory
-        _AddDllDirectory.argtypes = [c_wchar_p]
-        # Needed to initialize AddDllDirectory modifications
-        windll.kernel32.SetDefaultDllDirectories(0x1000)
```

Comment:
Presumably this line is the culprit

Reply:
Yes, but there seems to be agreement that that whole change was unneeded, so this does a more complete reversion.